Speech/src/DeprecatedTrait.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Speech;
+
+trait DeprecatedTrait
```

Comment:
I was thinking we would delete the Trait when it's no longer used too.

Reply:
In that case, we would want to mark the trait as deprecated as well I believe. I'm not a huge fan of introducing code in a deprecated state. Since it is such a small amount of code and we'll be removing it shortly thereafter I lean towards just directly implementing the calls.

Reply:
I already removed it, but I think if ever run into something like this again, we can mark it as `@internal` to declare that the artifact should be only used internal for package developers.